stubs/fpdf2/fpdf/drawing.pyi

```diff
@@ -1,18 +1,20 @@
 import decimal
 from _typeshed import Incomplete, Self
 from collections import OrderedDict
-from collections.abc import Generator
-from contextlib import _GeneratorContextManager
+from collections.abc import Generator, Iterator
+from contextlib import contextmanager
 from re import Pattern
-from typing import ClassVar, NamedTuple
+from typing import ClassVar, NamedTuple, TypeVar
 from typing_extensions import TypeAlias
 
 from .syntax import Name, Raw
 
 __pdoc__: dict[str, bool]
 
-def force_nodocument(item): ...
-def force_document(item): ...
+_T = TypeVar("_T", bound=Callable[Any])
```

Comment:
Maybe we should rename this to something like `_CallableT`. A common convention (at least within typeshed) is that `_T` can be anything, and bound types have the bound in the name.

Reply:
I've updated this.